analytics/java/src/main/java/com/koverse/examples/analytics/JavaTokenizerDataFrameTransform.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2016 Koverse, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.koverse.examples.analytics;
+
+import static org.apache.spark.sql.functions.callUDF;
```

Comment:
Please run optimize imports, there are a bunch of unused imports in this class

Reply:
done